Readme.md

```diff
@@ -193,6 +193,21 @@ gulp.task('watch', () => {
 ```
 
 
+## Using pnpm for package restore
+
+[pnpm](https://pnpm.io/) is a faster and more efficient package manager. If it's installed globally, then the Gulp Extension module will use that instead of *npm* to restore packages.
```

Comment:
```suggestion
[Pnpm](https://pnpm.io/) is a faster and more efficient package manager. If it's installed globally, then the Gulp Extension module will use that instead of *npm* to restore packages.
```